app/src/main/java/com/example/molliestephenson/tictactoe/MainActivity.java

```diff
@@ -2,12 +2,62 @@
 
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.widget.Button;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import ttt.Player;
+import ttt.game.Board;
+import ttt.game.GameEngine;
+import ttt.game.Marks;
 
 public class MainActivity extends AppCompatActivity {
 
+    private List<Button> mBoard = new ArrayList<>();
+    private GameEngine mGameEngine;
```

Comment:
What is the `m` in `mBoard` and `mGameEngine`? If its hungarian notation and stands for `member`, then please drop the `m`. We can tell that it is a member/field/instance variable of `MainActivty` just by where it is placed.